layers/osm.py

```diff
@@ -179,3 +179,41 @@ def roadHasBusLane(tags):
     # or any other users.
 
     return False
+
+def makeCrossings(
+    osm_input,
+):
+    if not osm_input:
+        raise Exception("You must specify --osm_input")
+
+    filename = "crossings"
+    tmp = f"tmp_{filename}"
+    ensureEmptyTempDirectoryExists(tmp)
+    osmFilePath = f"{tmp}/extract.osm.pbf"
+    run(
+        [
+            "osmium",
+            "tags-filter",
+            osm_input,
+            "n/crossing",
+            "-o",
+            osmFilePath,
+        ]
+    )
+    tmpGeojsonFilepath = f"{tmp}/{filename}.geojson"
+    outputFilepath = f"output/{filename}.geojson"
+    convertPbfToGeoJson(osmFilePath, tmpGeojsonFilepath, "point", includeOsmID=True)
+
+    def fixProps(inputProps):
+        outputProps = {}
+        try:
+            outputProps["osm_id"] = inputProps["@id"]
+            outputProps["crossing"] = inputProps["crossing"]
+        except:
+            # Ignore parsing errors and missing values
+            pass
+        return outputProps
+
+    cleanUpGeojson(tmpGeojsonFilepath, fixProps)
+
+    run(["cp", tmpGeojsonFilepath, outputFilepath])
```

Comment:
I think you need a `convertGeoJsonToPmtiles` too -- just london geojson is 5MB. Because it's just points as input, we _may_ need to pass extra flags like the autozoom `-zg` bit to make sure points show up. Easiest to just see in the frontend